ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -508,10 +518,63 @@ export default class TestDataModal extends React.Component {
               <Col className="p-2 col-4">
                 <InputFilter
                   disabled={relatedTests.length > 0}
+                  placeholder="filter tests e.g. linux tp5o"
                   updateFilterText={this.applyFilters}
                 />
               </Col>
             </Row>
+            {availableTags.length > 1 && (
+              <>
+                <Row className="justify-content-start">
+                  <Col className="p-2">
+                    <FormGroup>
+                      <Label for="selectMultiTags">Tags</Label>
+                      <Input
+                        className="fa"
+                        type="select"
+                        name="selectMultiTags"
+                        id="selectMultiTags"
+                        multiple
+                      >
+                        {availableTags.sort().map((tag) => (
+                          <option
+                            key={`available-tag-${tag}`}
+                            data-testid={`available-tag ${tag}`}
+                            onClick={() => this.toggleTag(tag)}
+                          >
+                            {tag}
+                          </option>
+                        ))}
+                      </Input>
+                    </FormGroup>
+                  </Col>
+                </Row>
+                <Row className="pb-2 justify-content-start">
+                  <Col className="p-2" sm="auto">
+                    {activeTags.sort().map((tag, index) => (
+                      <React.Fragment key={`active-tag-${tag}`}>
+                        <Button
+                          id={`active-tag-${index}`}
+                          data-testid={`active-tag ${tag}`}
+                          className="mr-3"
+                          outline
+                          onClick={() => this.toggleTag(tag)}
+                        >
+                          {tag}
+                          <Badge color="btn-darker-secondary">×</Badge>
```

Comment:
Wrapping the x inside of the badge makes it tiny, since some of the styling is only applied to that. What I meant with my suggestion was that the entire text would be wrapped in a badge. 


![Screen Shot 2020-08-17 at 10 27 34 AM](https://user-images.githubusercontent.com/19615783/90425316-67c4cb80-e074-11ea-9102-6e0e99a75bcf.png)

But you'll need to remove the button component and give the badge a button role and any other attributes to make it accessible.

Reply:
Also, the "btn-darker-secondary" is a class that overrides the color prop, so you need to use `className`.